src/test/java/com/google/devtools/build/lib/pkgcache/IncrementalLoadingTest.java

```diff
@@ -412,7 +413,7 @@ public void close() {
     private class ManualDiffAwarenessFactory implements DiffAwareness.Factory {
       @Nullable
       @Override
-      public DiffAwareness maybeCreate(Root pathEntry) {
+      public DiffAwareness maybeCreate(Root pathEntry, ImmutableSet<Path> ignorePaths) {
```

Comment:
nit: `s/ignorePaths/ignoredPaths`. Same for the rest of the occurrences.

Reply:
No problem! I had a reason for `ignorePaths` (even though I didn't like it) but I can't find it anymore. Oh well!